CDCgov/prime-reportstream
# Experience/13620/submissions filename

This PR ...

Adds the filename to the response of the Submission Table API

Test Steps:
1. Open Postman and run the following:` /api/waters/org/ignore/submissions?pageSize=61&cursor=3000-01-01T08%3A00%3A00.000Z&since=2000-01-01T08%3A00%3A00.000Z&until=3000-01-01T08%3A00%3A00.000Z&sortcol=undefined&sortdir=DESC`

2. Sign into RS as a sender
3. See filename in the submissions table

## Changes
- The response will now return the fileName and fileType
 ```
{
        "submissionId": 1885,
        "timestamp": "2024-03-14T17:40:50.641Z",
        "sender": "ignore.ignore-elr-elims",
        "httpStatus": 201,
        "id": "03c3b7ab-7c65-4174-bea7-9195cbb7ed01",
        "topic": "elr-elims",
        "reportItemCount": 1,
        "fileName": "None-03c3b7ab-7c65-4174-bea7-9195cbb7ed01-20240314174050.hl7",
        "fileType": "HL7"
    },
```


![Screenshot 2024-03-28 at 4 43 07 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/16de62c7-0da5-4fec-b42a-0a791d1107cc)

![Screenshot 2024-04-01 at 9 56 40 AM](https://github.com/CDCgov/prime-reportstream/assets/102491809/a0fd2438-a0b9-4936-bde2-096cc599607d)

## Checklist

### Testing

## Linked Issues
- Fixes #13620
